config.sample.js

```diff
@@ -134,7 +134,11 @@ module.exports = {
          /**
           * Surrounds all cimpler output in formatted html tags
           */
-         printHtml: true
+         printHtml: true,
+         /**
+          * Do not merge in the main branch
+          */
+         noMerge: true
```

Comment:
Since this is JS, we can technically add trailing commas if we like:
```suggestion
         noMerge: true,
```

Reply:
👍